resources/views/layouts/components/scripts.blade.php

```diff
@@ -0,0 +1,7 @@
+<script src="{{asset('js/plugins/jquery.min.js')}}"></script>
+<script src="{{asset('js/plugins/owl.carousel.min.js')}}"></script>
+<script src="{{asset('js/plugins/bootstrap.bundle.min.js')}}"></script>
+
+<script type="application/javascript" src="{{asset('js/main.js')}}"></script>
+<script type="application/javascript" src="{{asset('js/service-carousel.js')}}"></script>
+<script type="application/javascript" src="{{asset('js/testimonial-carousel.js')}}"></script>
```

Comment:
remove this two file and combine the content in one file **homepage.js**

Reply:
i move this file in one homepage.js